tests/APITests/APITest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Test\APITests;
+
+use App\Database\ConnectionManager;
+use App\Database\QueryBuilder;
+use App\Database\QueryWriterFactory;
+use App\Mapper\WordMapper;
+use App\Model\Word;
+use App\Repository\WordRepository;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\TestCase;
+
+class APITest extends TestCase
```

Comment:
APITest -> ApiTest (PascalCase/camelCase to snake_case rule of thumb)

Also ApiTest implies abstract case (testing whole api). I would suggest to make this class abstract and create new task (WordApiTest or WordTest or maybe better) that extends ApiTest. Also leave out common stuff in ApiTest case.